js/src/momoa.d.ts

```diff
@@ -0,0 +1,113 @@
+
+//-----------------------------------------------------------------------------
+// Options
+//-----------------------------------------------------------------------------
+
+type MomoaMode = "json" | "jsonc";
+
+interface MomoaTokenizeOptions {
+    mode: MomoaMode;
+    ranges: boolean;
+}
+
+interface MomoaParseOptions {
+    mode: MomoaMode;
```

Comment:
I think you can say `Readonly<{ mode ..... }>` if you don't want to add the keyword on front of every line.

Reply:
Indeed! My example is mainly to also make maybe some props read-only and others not depending on the use. In React I tend to make all read-only to avoid mutations.